akka-persistence-typed/src/test/scala/docs/akka/persistence/typed/InDepthPersistentBehaviorSpec.scala

```diff
@@ -94,14 +94,15 @@ object InDepthPersistentBehaviorSpec {
   //#post-added-command-handler
 
   //#by-state-command-handler
-  private def commandHandler: CommandHandler[BlogCommand, BlogEvent, BlogState] = CommandHandler.byState {
-    case state if state.isEmpty  ⇒ initial
-    case state if !state.isEmpty ⇒ postAdded
-  }
+  private val commandHandler: CommandHandler[BlogCommand, BlogEvent, BlogState] =
```

Comment:
Do we want to use aliases in the docs?

Reply:
I changed to the full signature. It's rather huge though:
```
private val commandHandler: (ActorContext[BlogCommand], BlogState, BlogCommand) ⇒ Effect[BlogEvent, BlogState]
```